doc/src/learn/architecture/validators.md

```diff
@@ -31,7 +31,7 @@ As with transactions, we note that the process of sharing a certificate with val
 
 ## The role of Narwhal and Tusk
 
-Sui takes advantage of [Narwhal and Tusk: A DAG-based Mempool and Efficient BFT Consensus](https://arxiv.org/pdf/2105.11827.pdf). Narwhal/Tusk (N/T) are also being developed by [Mysten Labs](https://mystenlabs.com/) so that per the referenced white paper, “When full agreement is required we use a high-throughput DAG-based consensus, e.g. [9] to manage locks, while execution on different shared objects is parallelized.”
+Sui takes advantage of [Narwhal and Tusk: A DAG-based Mempool and Efficient BFT Consensus](consensus.md). Narwhal/Tusk (N/T) are also being developed by [Mysten Labs](https://mystenlabs.com/) so that when full agreement is required we use a high-throughput DAG-based consensus to manage locks while execution on different shared objects is parallelized.
```

Comment:
full agreement -> Byzantine agreement

Reply:
developed by -> implemented by

Reply:
to mange locks -> to manage shared locks

Reply:
mmh parallelising the execution of different shared objects has little  to do with consensus and the locks (this sentence could be confusion)

Reply:
Hi Alberto,

Can you create a PR with your changes and have Francois review it? This is a link swap PR. :-)

Reply:
> Hi Alberto,
> 
> Can you create a PR with your changes and have Francois review it? This is a link swap PR. :-)

I took care of all of these changes except for the last, which isn't actionable. I don't even know what sentence this refers too.

So I filed this new issue for you and Francois to review and edit the entire page as a team:
https://github.com/MystenLabs/sui/issues/1638

Thanks!